pages/common/kdig.md

```diff
@@ -0,0 +1,24 @@
+# kdig
+
+> Advanced DNS lookup utility.
+> More information: <https://manned.org/kdig>.
```

Comment:
I think the author's website is better in this case
```suggestion
> More information: <https://www.knot-dns.cz/docs/latest/html/man_kdig.html>.
```

Reply:
agree